docker/test/integration/Dockerfile

```diff
@@ -1,12 +1,19 @@
 FROM ubuntu:18.04
+# yandex/clickhouse-integration-test
 
-RUN apt-get update && apt-get -y install tzdata python llvm-6.0 llvm-6.0-dev
+RUN apt-get update \
+    && env DEBIAN_FRONTEND=noninteractive apt-get -y install tzdata python llvm-6.0 llvm-6.0-dev libreadline-dev libicu-dev bsdutils \
```

Comment:
not sure if python is really needed here

Reply:
I'm also not sure :)